release-notes/release-notes55.md

```diff
@@ -122,6 +122,18 @@ For information on the    plugin, see: []().
 * Fixed `editor.selection.setCursorLocation` incorrectly placing the cursor outside `pre` elements in some circumstances.
 * Fixed an exception being thrown when pressing the enter key inside pre elements while `br_in_pre` setting is false.
 
+## Deprecated features
+
+The following features have been deprecated with the release of {{site.productname}} 5.5:
+
+- [The premium Mexican Spanish language pack has been deprecated](#thepremiummexicanspanishlanguagepackhasbeendeprecated).
+
+### The premium Mexican Spanish language pack has been deprecated
```

Comment:
`Spanish (Mexico) language pack` maybe? as that's it's actual name, as I'm worried this isn't completely clear.

Reply:
went with "Mexican Spanish" because that's the name of the language?

Reply:
It might be the name of the language, but it's not the name of the language pack and we don't call it this anywhere else.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1693/commits/db091fed01d8c89f5d3c265d221a4c8c7ac60858 and https://github.com/tinymce/tinymce-docs/pull/1693/commits/4140310556b748123a80e1b1c5f22f637dfdefe2